package-lock.json

```diff
@@ -7432,19 +7695,13 @@
       "integrity": "sha512-IWCo7XexAtoDx0At16UH7ySejlxcQqg+thB1TUwCti3F5b8MSjIT+mhLaJI6qvjEzSPs56WqKMJKeo8Y1ikIyw==",
       "dev": true,
       "requires": {
-        "esprima": "^1.2.2",
+        "esprima": "^4.0.1",
         "estraverse": "^1.5.1",
         "minimatch": "^3.0.4",
         "through": "^2.3.4",
         "type-name": "^1.0.0"
       },
       "dependencies": {
-        "esprima": {
-          "version": "1.2.5",
-          "resolved": "https://registry.npmjs.org/esprima/-/esprima-1.2.5.tgz",
-          "integrity": "sha1-CZNQL+r2aBODJXVvMPmlH+7sEek=",
-          "dev": true
-        },
```

Comment:
This is the key change that differentiates #80 from #79.
:grimacing: it's pretty gross but it fixes the build!